Resources/Prototypes/_NF/Entities/Objects/Weapons/Guns/Ammunition/Magazines/novalitec1clip.yml

```diff
@@ -0,0 +1,48 @@
+# Empty mags
+- type: entity
+  id: BaseMagazineNovaliteC1
+  name: "High Capacity Assault Clip (.20 rifle)"
+  parent: BaseItem
+  abstract: true
+  components:
+  - type: Tag
+    tags:
+      - MagazineNovaliteC1
+  - type: Item
+    size: Small
+  - type: BallisticAmmoProvider
+    mayTransfer: true
+    whitelist:
+      tags:
+        - CartridgeRifle
+    capacity: 15
+  - type: ContainerContainer
+    containers:
+      ballistic-ammo: !type:Container
+  - type: Sprite
+    sprite: _NF/Objects/Weapons/Guns/Ammunition/Magazine/Rifle/novalite_clip.rsi
+    layers:
+    - state: base
+      map: ["enum.GunVisualLayers.Base"]
+    - state: mag-1
+      map: ["enum.GunVisualLayers.Mag"]
+  - type: MagazineVisuals
+    magState: mag
+    steps: 16
+    zeroVisible: false
+  - type: Appearance
+
+# Magazines
+- type: entity
+  id: MagazineNovaliteC1
+  name: "High Capacity Assault Clip (.20 rifle)"
```

Comment:
'magazine' instead of 'clip' please

Reply:
Renamed to Novalite En Bloc. That appears to be the historically correct name for this type of ammunition loading mechanism.